pkg/nethelper/ipcalc.go

```diff
@@ -1,6 +1,8 @@
-// This is an adaptation of below gist:
-//
-//  https://gist.github.com/kotakanbe/d3059af990252ba89a82
+/*
```

Comment:
Undo this change and just add an empty line above `package nethelper`

Reply:
Made the changes please see to it.